tos/chips/cortex/m4/sam4/l/ast/HplSam4lASTP.nc

```diff
@@ -203,6 +203,7 @@ implementation
 
 	async command uint32_t HplSam4lAST.getAlarm()
 	{
+	    while(AST->sr.bits.busy == 1);
```

Comment:
On Page 469 of the datasheet it says: "When the SR.BUSY bit is set writes to this register will be discarded and this register will read as zero." The phrase "this register" refers to the "ar0" register.

In setAlarm, the code correctly spin loops until busy is not set. In getAlarm, I didn't see this. Furthermore, when I call getAlarm, I got 0. So, I added this fix.